content/guides/virtualizing/vultr.md

```diff
@@ -0,0 +1,43 @@
++++
+type = "article"
+title = "Setting up Haiku in Vultr"
+date = "2019-12-21T19:04:00.000Z"
+tags = []
++++
+
+Vultr allows one to upload a custom iso on their instances.
+With this, many new possibilities are open - such as setting up a buildbot, automated testing, benchmarking and more.
+This task assumes you have a Vultr account and that you are aware that it will charge you for setting up and running an instance. 
+
+##Preparing a direct ISO link
+
+Vultr doesn't allow one to upload a custom iso from a local machine. Instead, you need a direct web download link. At the moment, Haiku does not provide these, so you will have to prepare one by yourself. Don't worry - it's not difficult. You can use a variety of services like Amazon S3, Dropbox, etc. Please check Vultr's documentation for more [information]("https://www.vultr.com/docs/requirements-for-uploading-an-os-iso-to-vultr#Where_can_I_upload_my_ISO")
+This guide will show you how to use Dropbox to provide a direct link for Vultr. You will need a Dropbox account for this. 
+
+##Using dropbox to serve a Haiku ISO for Vultr
+
+First, download a Haiku anyboot zip of your choice. Both x32 and x64 work. Unzip it, and upload the ISO file to your Dropbox. After it is done uploading, select it and press share. Near the bottom right-hand side, click "Create link".
+![Dropbox Share](/files/guides/virtualizing/vultr/dropbox-share.png)
+Copy the link that appears. Replace the dropbox.com part of it with dl.dropboxusercontent.com, and remove everything after the final "?".
+From example: change "https://www.dropbox.com/s/h0h3id5sfw2t5esk/haiku-release-anyboot.iso?dl=0" to "https://www.dl.dropboxusercontent.com/s/h0h3id5sfw2t5esk/haiku-release-anyboot.iso"
```

Comment:
For example links, it can be redacted here to show it is an example link.
```suggestion
From example: change "https://www.dropbox.com/s/h0h3id5sfw2t5esk/haiku-release-anyboot.iso?dl=0" to "https://www.dl.dropboxusercontent.com/s/xxxxxxxxxxxxxxx/haiku-release-anyboot.iso"
```